### DISCUSSION TITLE
Universal Display Names

### DISCUSSION THREAD

**Discussion by PapierTanne210** [nextcord/nextcord#1066] [Ideas]
Unfortunately, Nextcord (Discord.PY) does not currently provide a built-in method to retrieve this universal display name directly. The available display_name property of the nextcord.User object returns the user's display name based on the current server context, which can be problematic, especially if the display name contains emojis. This affects images, for example.
So it would be very nice if you could add this permanently to make the universal display names usable!

**Comment by spifory** [comment]
This is a duplicate of #1060.

**Reply by PapierTanne210** [reply]
Sorry, I must have overseen this
